DB設計

yoshinoayumu opened the pull request:
# what
**tables**
users,items,purchases,address
Association

yoshinoayumu committed:
DB設計

tables
users,items,purchases,address

Association

yoshinoayumu committed:
fix

kata-myu commented on README.md:
```diff
@@ -1,24 +1,84 @@
+
+
+
+## addresses table
+| Column                     | Type    | Options                 |
```
こちらのaddressesテーブルですが、itemsテーブルと重なってしまっています。
見本のアプリなどを参考にして必要なカラムなどを確認してみましょう。

kata-myu reviewed (commented):
レビュー依頼ありがとうございます。
コメントいたしましたので、ご確認をお願いいたします。

yoshinoayumu committed:
addressesテーブルfix

column変更

yoshinoayumu commented:
addressesテーブルの修正を行いました。何卒よろしくお願い致します！

yoshinoayumu committed:
fix

building option

yoshinoayumu committed:
fix

郵便番号・電話番号はinteger型ではなく、string型を使用。
　①郵便番号に-が含まれている
　②郵便番号・電話番号共に0から始まるケースがある。
integer型だと、最初の0が省略されてしまい、正しい値にならないため

yoshinoayumu commented:
修正いたしました。ご確認何卒よろしくお願い致します！

kata-myu reviewed (commented):
レビュー依頼ありがとうございます！
LGTMとさせていただきます。実装お疲れ様でした。

マージ後、ローカルのmasterブランチに忘れずにプルしましょう。
その後、新しいブランチを作成し次の実装に取り組んでいただけますと幸いです。
引き続き実装頑張っていきましょう！


↑（LGTMのときはこのようなテンプレを使っています）
